polynote-runtime/src/main/scala/polynote/runtime/ValueRepr.scala

```diff
@@ -233,6 +240,10 @@ object StreamingDataRepr {
   private[polynote] def getHandle(handleId: Int): Option[Handle] = Option(handles.get(handleId))
   private[polynote] def releaseHandle(handleId: Int): Unit = getHandle(handleId).foreach(_.release())
 
+  // visible for testing
+  private[polynote] def addHandle(handle: Handle): Unit =
+    handles.put(handle.handle, handle)
```

Comment:
If we're testing with this method should we use it instead of `handles.put`? 
Otherwise, maybe it would be clearer to just make `handles` itself accessible to the tests??

Reply:
I didn't actually end up using this for testing, so I'll remove it.

Reply:
Fixed